src/logic/formatters.js

```diff
@@ -11,19 +11,19 @@ export const cutoffToString = (cutoff, eventId) => {
   }
 };
 
-export const timeLimitToString = timeLimit => {
+export const timeLimitToString = (timeLimit, globalTranslation) => {
```

Comment:
Let's pass it as options and have a default.

```suggestion
export const timeLimitToString = (timeLimit, options = {}) => {
  const { totalText = "total" } = options;
```

Note that we need to pass it in the scorecard code, which we currently don't.